<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb100">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb500">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoffe">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffe(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTea">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMilk">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
